ggv_generator.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import pandas as pd
+import itertools
+import engine
+import multiprocessing
+from tqdm import tqdm
+from time import perf_counter
+
+def ggv_generator(vehicle_params, sweep_ranges:dict, mesh:int):
```

Comment:
Can we use a more specific term than mesh (this can refer to a lot of things, and most meshes aren't really even rectangular) - confusing if you're just using it to define the spacing on the linspace. Nitpicking but would prefer more DOE oriented language (i.e. factorial search with 10 levels per factor).

Reply:
IDK what that means and IDK if anyone else does